PWGHF/Tasks/taskXicc.cxx

```diff
@@ -54,6 +54,7 @@ struct HfTaskXicc {
     registry.add("hChi2PCA", "#Xi^{++}_{cc} candidates;chi2 PCA (cm);entries", {HistType::kTH2F, {{500, 0., 0.01}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
     registry.add("hd0Prong0", "#Xi^{++}_{cc} candidates;prong 0 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -0.02, 0.02}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
     registry.add("hd0Prong1", "#Xi^{++}_{cc} candidates;prong 1 DCAxy to prim. vertex (cm);entries", {HistType::kTH2F, {{200, -0.02, 0.02}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hd0d0", "#Xi^{++}_{cc} candidates;product of DCAxy to prim. vertex (cm^{2}); #it{p}_{T} (GeV/#it{c})", {HistType::kTH2F, {{500, -0.05, 0.05}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
```

Comment:
Missing `entries` as the label of the z axis.

Reply:
It is a TH2 so there is not label on z axis. On x axis there is the impact parameter product, while on the y axis the transverese momentum. This is correct.

Reply:
1D histogram has 2 axes: x for the variable, y for the entries.
2D histogram has 3 axes: x, y for the variables, z for the entries.
Do we agree?

Reply:
No, I don't agree...I have always defined TH2F as this. In the end you have a bi-dimensional histogram and what matters is what you have on the x and y axes, because they are displayed.

Reply:
You always display the z axis too. You cannot read the bin values otherwise.
See [the TH2 documentation](https://root.cern/doc/master/classTH2.html#a5667b0486e50fd40cba6b640ca4ba67b) for example.
> [in] | title | histogram title. If title is of the form stringt;stringx;stringy;stringz the histogram title is set to stringt, the x axis title to stringx, the y axis title to stringy, etc.

Reply:
Again this is not true and incorrect. I can read the bin content...what is reported in the documentation of TH2 is a general example of how you can define "title" in a histogram. I just tried to create a TH2 without setting the z-axis label and it works perfectly, so I won't add this because it is useless

Reply:
I added entries even if absolutely not needed...now it should be merged

Reply:
Thanks a lot @strogolo and sorry for the misunderstanding!